frontend/test/metabase/scenarios/question/settings.cy.spec.js

```diff
@@ -147,6 +151,93 @@ describe("scenarios > question > settings", () => {
       }
     });
 
+    // https://github.com/metabase/metabase/pull/21338#pullrequestreview-928807257
+    it("should allow columns ordering after column removal and addition via sidebar (metabase#13455)", () => {
```

Comment:
I'll have to take a deeper look, but it seems that this test is basically the exact copy of the previous scenario with the extra step. In that case, let's expand the original test rather than adding another one.

https://docs.cypress.io/guides/references/best-practices#Creating-tiny-tests-with-a-single-assertion

Reply:
Yeah, that's exactly what it does. I'll combine the new test with the previous scenario. I didn't do that in the beginning because I'm afraid I'll make the test bloated. But if that's what Cypress is suggesting then I'd stick to it.

Reply:
Fixed